src/main/java/org/mamute/sanitizer/HtmlElement.java

```diff
@@ -14,10 +15,12 @@
 public class HtmlElement {
 
 	private String element;
+	private ElementPolicy elementPolicy;
```

Comment:
good point - I think I have to go with guava's Optional since Mamute does not depend on Java 8 yet.